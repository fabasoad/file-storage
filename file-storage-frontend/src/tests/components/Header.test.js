import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('should render Header correctly', () => {
    const title = 'abc123';
    const subtitle = 'def456';
    const wrapper = shallow(<Header title={title} subtitle={subtitle} />);
    expect(wrapper).toMatchSnapshot();
});