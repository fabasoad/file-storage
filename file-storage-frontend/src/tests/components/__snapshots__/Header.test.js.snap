// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="header__title"
    >
      abc123
    </h1>
    <h2
      className="header__subtitle"
    >
      def456
    </h2>
  </div>
</div>
`;
